### ISSUE TITLE
Architecture differences in Zoo Models ?

### ISSUE THREAD

**Issue by nitin2212** [pytorch/vision#2286]
Hi, I am comparing few zoo models implementation in DL4J with Pytorch zoo models and found
that the padding in Convolution layers does not match most of time ?

For **Resnet50 and SqueezeNet** :

In DL4J, they **do not** apply padding : [0, 0] ; while in PyTorch they have padding [1, 1].
This results in different output in layers

In DL4J, they apply **Bias** in Conv layers while in PyTorch, they do not.

Why such irregularities in Network structure across frameworks ?

**Comment by fmassa** [comment]
Hi,

ResNets use padded convolution, so that when having a kernel size of 3x3 the output size is the same. Maybe they didn't use `[0, 0]`, but passed something like `padding_type='SAME'`?

About the `bias` in convolutional layers, they are unnecessary because there is a BatchNormalization just after every convolution, making the bias in the convolution redundant.
